sdk/sdk-common/src/swap/SimulatedSwapData.ts

```diff
@@ -0,0 +1,12 @@
+import { Percentage } from '@summerfi/sdk-common/common'
+import { QuoteData } from './QuoteData'
+
+/**
+ * Represents the data returned for each Swap in simulation.
+ * It is derived from the `QuoteData` type with the `estimatedGas` field omitted,
+ * as gas estimation is not relevant for simulation purposes.
+ */
+export type SimulatedSwapData = Omit<QuoteData, 'estimatedGas'> & {
+  slippage: Percentage
```

Comment:
we also need to get swap Fee and Price impact for each swap, could you investigae if that's possible to get? I was expecting it was in the quote no?

Reply:
Sure thing